src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -209,22 +213,24 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
                                                                                const ByteSpan & dacCertDerBuffer,
                                                                                const ByteSpan & attestationNonce)
 {
+    VendorId dacVendorId = VendorId::NotSpecified;
     // match DAC and PAI VIDs
     if (!paiCertDerBuffer.empty())
     {
-        VendorId paiVid;
-        VendorId dacVid;
+        uint16_t paiVid = VendorId::NotSpecified;
+        uint16_t dacVid = VendorId::NotSpecified;
 
-        CHIP_ERROR error     = ExtractVIDFromX509Cert(paiCertDerBuffer, paiVid);
+        CHIP_ERROR error     = ExtractDNAttributeFromX509Cert(MatterOid::kVendorId, paiCertDerBuffer, paiVid);
         const bool paiHasVid = error != CHIP_ERROR_KEY_NOT_FOUND;
```

Comment:
`VID` is a mandatory (not optional) field in the PAI certificate.

Reply:
Addressed in 92dd5d5